package.json

```diff
@@ -43,92 +46,78 @@
       "title": "SonarLint",
       "properties": {
         "sonarlint.testFilePattern": {
-          "type": [
-            "string",
-            "null"
-          ],
+          "type": "string",
           "default": "{**/test/**,**/*test*,**/*Test*}",
-          "description": "Files whose name match this glob pattern are considered as test files"
+          "markdownDescription": "Files whose name match this [glob pattern](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob) are considered as test files by analyzers. Most rules are *not* evaluated on test files."
         },
         "sonarlint.analyzerProperties": {
-          "type": [
-            "object",
-            "null"
-          ],
+          "type": "object",
           "patternProperties": {
             "^.*$": {
               "type": "string",
-              "description": "One entry value"
+              "markdownDescription": "One entry value"
             }
           },
-          "default": null,
-          "description": "Extra properties that will be passed to the analyzers. For example {\"sonar.javascript.globals\": \"xxx\"}. See documentation of each analyzers."
+          "markdownDescription": "Extra properties that could be passed to the code analyzers. e.g. `{\"sonar.javascript.globals\": \"xxx\"}`. See [documentation](https://redirect.sonarsource.com/doc/plugin-library.html) of each analyzers."
         },
         "sonarlint.disableTelemetry": {
           "type": "boolean",
           "default": false,
-          "description": "Disable sending anonymous usage statistics"
+          "markdownDescription": "Disable sending anonymous usage statistics"
         },
         "sonarlint.ls.javaHome": {
-          "type": [
-            "string",
-            "null"
-          ],
-          "default": null,
-          "description": "Specifies the folder path to the JRE (8 or more recent) used to launch the SonarLint Language Server.\nOn Windows, backslashes must be escaped, i.e.\n\"sonarlint.ls.javaHome\":\"C:\\\\Program Files\\\\Java\\\\jdk1.8.0_161\"",
+          "type": "string",
+          "markdownDescription": "Path to a JRE (8 or more recent) used to launch the SonarLint Language Server. \n\n On Windows, backslashes must be escaped, e.g. `C:\\\\Program Files\\\\Java\\\\jdk1.8.0_161`",
```

Comment:
I'm wondering if using `/` would work. I would much prefer writing like that. I can test in Windows tomorrow, if you like.

Reply:
Please test and tell me. But I found other extensions doing the same, so I have doubt.

Reply:
Tested, using `/` works on Windows. Even in earlier versions of VSCode (I found my settings on Windows were always using `/`).

Anyway, let's keep it as it is. Perhaps more natural with `\` for Windows users, and as you said other plugins are doing the same.